README.md

```diff
@@ -2,29 +2,34 @@
 
 [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors)
 
-## What is `FLINT`?
+## What is FLINT?
 
-The Full Lands Integration Tool (FLINT) is a C++ framework that combines satellite and ground data in ways that meet policy needs. It is based on over 20 years of experience building and operating integration tools in Australia and Canada.   It's an Open-source Library maintained under [**moja.global**](https://github.com/moja-global/About-moja-global/blob/master/README.md), a project under the Linux Foundation
+Full Lands Integration Tool (FLINT) is a C++ framework that combines satellite and ground data  in a way that meets policy requirements.It is based on more than 20 years of experience in building and operating integrated tools in Australia and Canada. It's an Open-Source Library maintained under [**moja global**](https://github.com/moja-global/About-moja-global/blob/master/README.md), a project under the Linux Foundation.
```

Comment:
There is an extra space between `data` and `in`. Use a Markdown beautifier tool to mitigate these issues.